app/javascript/react/components/GameShowTile.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+
+const GameShowTile = props => {
+  const name = props.name
+  const playerNum = props.playerNum
+  const description = props.description
```

Comment:
not required, but I would strongly suggest props desturcturing that is closer to common practice 

`const GameShowTile = ({ name, playerNum, description }) => {`